src/accounts/context/userAccount.tsx

```diff
@@ -88,10 +88,6 @@ export const UserAccountProvider: FC<Props> = React.memo(({children}) => {
       if (resp.status !== 204) {
         throw new Error(resp.data.message)
       }
-
-      if (!isFlagEnabled('trackCancellations')) {
-        history.push(`/logout`)
```

Comment:
What’s the different between postCancel and postSignout?

Reply:
I think `postCancel` handles the cancellation of Subscription and `postSignout` is just responsible to sign out(oss/cloud). Also, `postCancel` is in Quartz and `postSignout` is in idpe.